libr/util/strbuf.c

```diff
@@ -71,10 +71,10 @@ R_API bool r_strbuf_reserve(RStrBuf *sb, int len) {
 	return true;
 }
 
-R_API bool r_strbuf_setbin(RStrBuf *sb, const ut8 *s, int l) {
+R_API bool r_strbuf_setbin(RStrBuf *sb, const ut8 *s, size_t l) {
 	r_return_val_if_fail (sb && s, false);
-	r_return_val_if_fail (l >= 0, false);
-
+	sb->len = l;
+	sb->ptrlen = l;
```

Comment:
mmm this smells odd. 
1) both len and ptrlen are set again before returning from this function
2) sb->ptrlen is used at line 80, but the condition `l + 1 > sb->ptrlen` would always be true with this.

Reply:
Yes you are right, this looks broken.